### PR TITLE
Merge docs into main

### PR DESCRIPTION
**Pull request by yaseminbridges**
Add @souzadevinicius docs files to main - getting the two branches on par so that we can delete the docs branch

Changed the branch for the docs deployment to the `main` branch

### CODE REVIEW

**Comment by yaseminbridges** [comment]
@souzadevinicius will need to review these files as he produced them to make sure that they are good to go to merge into the `main` branch

File: resources/custom.Makefile
Comment (souzadevinicius) [inline resources/custom.Makefile:1]: Why percent sign was removed?

**Review by souzadevinicius — APPROVED** [review]
Just one quick question

File: resources/custom.Makefile
Comment (yaseminbridges) [reply resources/custom.Makefile:1]: I am not sure, I didn't touch these files. It must be something you have previously changed in the main branch that doesn't match the docs. Please change it if it's not correct

**Review by souzadevinicius — APPROVED** [review]

File: mkdocs.yml [hunk 1]
```diff
@@ -43,6 +44,9 @@ watch:
 
 nav:
   - "index.md"
+  - Pipeline:
+    - "exomiser_pipeline.md"
```
Comment (matentzn) [inline mkdocs.yml:48]: Doesn't this have to be  key value pairs? How would mkdocs know otherwise how to render the link?

Comment (souzadevinicius) [reply mkdocs.yml:48]: The link will be based on the file's name and the nav title is the file header content.

**Review by matentzn — APPROVED** [review]

**Pull request merged**